matlabCode/createBIDS_ieeg_json.m

```diff
@@ -67,82 +83,79 @@
 % the equipment from being identifiable, as long as each pseudonym is unique 
 % within the dataset.
 
+ieeg_json.ECOGChannelCount = ''; % Number of iEEG surface channels included in the recording (e.g. 120) 
 
+ieeg_json.SEEGChannelCount = ''; % Number of iEEG depth channels included in the recording (e.g. 8)
 
-%% General fields, shared with MEG BIDS:
-
-
-
-%%  Required fields:
-
-ieeg_json.EEGChannelCount = ''; % Number of EEG channels included in the recording (e.g. 0) 
+ieeg_json.EEGChannelCount = ''; % Number of scalp EEG channels recorded simultaneously (e.g., 21)
 
-ieeg_json.EOGChannelCount = ''; % Number of EOG channels included in the recording (e.g. 1)
+ieeg_json.EOGChannelCount = ''; % Number of EOG channels
 
-ieeg_json.ECGChannelCount = ''; % Number of ECG channels included in the recording (e.g. 1)
+ieeg_json.ECGChannelCount = ''; % Number of ECG channels
 
-ieeg_json.EMGChannelCount = ''; % Number of EMG channels included in the recording (e.g. 1)
+ieeg_json.EMGChannelCount = ''; % Number of EMG channels
 
-ieeg_json.MiscChannelCount = ''; % Number of miscellaneous channels included in 
-% the recording (e.g. 1)
+ieeg_json.MiscChannelCount = ''; % Number of miscellaneous analog channels for auxiliary  signals
 
-ieeg_json.TriggerChannelCount = ''; % Number of channels for digital (TTL bit level) 
-% triggers (e.g. 0) 
+ieeg_json.TriggerChannelCount = ''; % Number of channels for digital (TTL bit level) triggers. 
 
-ieeg_json.PowerLineFrequency = ''; % Frequency (in Hz) of the power grid where 
-% the iEEG recording was done (i.e. 50 or 60) 
-
-
-
-%% Optional fields:
 ieeg_json.RecordingDuration = ''; % Length of the recording in seconds (e.g. 3600)
 
-ieeg_json.RecordingType = ''; % continuous, epoched 
+ieeg_json.RecordingType = ''; % continuous or epoched 
```

Comment:
comment should be: Defines whether the recording is  “continuous” or  “epoched”; this latter limited to time windows about events of interest (e.g., stimulus presentations, subject responses etc.)